### PR TITLE
IOS #30 Implement slide out navigation drawer

### PR DESCRIPTION
**Pull request by raymondjacobson**
Graphic content, menu items, and view controllers for other views created via storyboard

### CODE REVIEW

File: whereuat-ios/Views/DrawerView/DrawerViewController.swift [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+//
+//  Created by Raymond Jacobson on 5/11/16.
+//  Copyright © 2016 whereuat. All rights reserved.
+//
+
+import UIKit
+import SlideMenuControllerSwift
+
+enum Menu: Int {
+    case Home = 0
```
Comment (kangp3) [inline whereuat-ios/Views/DrawerView/DrawerViewController.swift:13]: Get rid of = 0

File: whereuat-ios/Views/DrawerView/DrawerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        self.selfNameView.backgroundColor = ColorWheel.transparent
+        self.selfNameView.textColor = ColorWheel.darkGray
+        self.selfNameView.font = FontStyle.h4
+        self.selfNameView.text = AppDrawer.selfName
+        
+        self.view.addSubview(self.selfNameView)
+    }
+    
+    /*
+     * drawSelfNameView draws the current account phone to the drawer
```
Comment (kangp3) [inline whereuat-ios/Views/DrawerView/DrawerViewController.swift]: drawSelfNumberView

**Comment by kangp3** [comment]
lgtm after comments

File: whereuat-ios/Views/DrawerView/DrawerViewController.swift (hunk 1 above)
Comment (raymondjacobson) [inline whereuat-ios/Views/DrawerView/DrawerViewController.swift:13]: "technically less expensive" so we will go with it. THANKS PETE!

**Comment by kangp3** [comment]
Also, does this close #30?

**Comment by raymondjacobson** [comment]
Closes #30

**Pull request merged**